packages/fuel-gauge/src/predicate/predicate-duplication.test.ts

```diff
@@ -0,0 +1,362 @@
+import { WalletUnlocked } from '@fuel-ts/account';
+import { launchTestNode } from 'fuels/test-utils';
+
+import { PredicateWithConfigurable } from '../../test/typegen';
+import type {
+  PredicateWithConfigurableConfigurables,
+  PredicateWithConfigurableInputs,
+} from '../../test/typegen/predicates/PredicateWithConfigurable';
+
+import { fundPredicate, assertBalance } from './utils/predicate';
+import { processPredicateData } from './utils/predicate/processPredicateData';
+
+/**
+ * @group node
+ * @group browser
+ */
+describe('Predicate.fromInstance', () => {
+  const amountToPredicate = 300_000;
+
+  const defaultValues = {
+    FEE: 10,
+    ADDRESS: '0x38966262edb5997574be45f94c665aedb41a1663f5b0528e765f355086eebf96',
+  };
+
+  describe('Basic Instance Operations', () => {
+    it('creates new predicate instance from existing one with default values', async () => {
+      using launched = await launchTestNode();
+
+      const provider = launched.provider;
+      const basePredicate = new PredicateWithConfigurable({
+        provider,
+        data: [defaultValues.FEE, defaultValues.ADDRESS],
+        configurableConstants: defaultValues,
+      });
+
+      const newPredicate: PredicateWithConfigurable = basePredicate.toNewInstance();
+
+      expect(newPredicate.predicateData).toEqual(basePredicate.predicateData);
+      expect(newPredicate.interface).toEqual(basePredicate.interface);
+      expect(newPredicate.provider).toEqual(basePredicate.provider);
+      expect(newPredicate.bytes).toEqual(
+        processPredicateData(basePredicate.initialBytecode, basePredicate.interface.jsonAbi, {})
+          .predicateBytes
+      );
```

Comment:
Since no values were used when creating the new instance:

```suggestion
      expect(newPredicate.bytes).toEqual(basePredicate.bytes);
```